clang/test/Driver/sycl-device-lib-old-model.cpp

```diff
@@ -378,3 +378,40 @@
 // RUN: -Xarch_device -fsanitize=memory -### 2>&1 | FileCheck %s -check-prefix=SYCL_DEVICE_LIB_MSAN_CPU
 // SYCL_DEVICE_LIB_MSAN_CPU: llvm-link{{.*}} "-only-needed" "{{.*}}libsycl-crt.bc"
 // SYCL_DEVICE_LIB_MSAN_CPU-SAME: "{{.*}}libsycl-msan-cpu.bc"
+
+/// ###########################################################################
+/// test behavior of libsycl-tsan.bc linking when -fsanitize=thread is available
+// RUN: %clangxx -fsycl --no-offload-new-driver %s --sysroot=%S/Inputs/SYCL -fsanitize=thread -### 2>&1 \
```

Comment:
Are these tests supported in `clang_cl` mode?

Reply:
Device sanitizer is not supported for windows.